public/js/app.js

```diff
@@ -18,7 +18,7 @@ const zoomToLocation = () => {
   }
 
   if (navigator.geolocation) {
-    showLoader();
+    //showLoader();
```

Comment:
is this supposed to be commented out or by mistake? if you meant to comment out, i prefer you delete it and the function if no one else calls it

Reply:
I will delete